modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -1,18 +1,21 @@
 // Module included in the following assemblies:
 //
-// * networking/ingress-controller-configuration.adoc
+// * networking/ingress-operator.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-autoscaling-ingress-controller_{context}"]
 = Autoscaling an Ingress Controller
 
-Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements, such as the requirement to increase throughput.
+
+The following procedure provides an example for scaling up the default Ingress Controller.
 
 .Prerequisites
-. You have the OpenShift CLI (`oc`) installed.
-. You have access to an {product-title} cluster as a user with the `cluster-admin` role.
-. You have the Custom Metrics Autoscaler Operator installed.
-. You are in the `openshift-ingress-operator` project namespace.
+
+* You have the {oc-first} installed.
```

Comment:
Thank you very much for providing preview links to the different distros, because it helped me notice that {oc-first} does not resolve to anything on the ROSA and OSD distros. Actually might be quite problematic beyond just this page if this attribute is used anywhere that's also included in those distros

Reply:
Good observation. I asked in our docs troubleshooting channel.

Reply:
I asked a question in the [forum-sd-docs ](https://redhat-internal.slack.com/archives/C0308P3FC7R/p1724775647719119)channel.

Reply:
No clear answer about this so reverting to the hardcoded value.